src/components/attachments/attachments.html

```diff
@@ -0,0 +1,33 @@
+<!-- Generated template for the AttachmentsComponent component -->
+
+<ion-card class="thumbnail-card" [ngSwitch]="true">
+  <img
+    [src]="url"
+    *ngSwitchCase="extension==='jpg'
```

Comment:
check for all formats of image. Refer imageListingcomponent.

Reply:
for image only three images extension is there jpg,png,jpeg
for video and audio added all other extension